### PR TITLE
{cae}[dummy] Samcef 17.0-03 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
requires https://github.com/hpcugent/easybuild-easyblocks/pull/678

### CODE REVIEW

**Comment by wpoely86** [comment]
looks fine

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b2395ca2068d8985f2e9 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

File: easybuild/easyconfigs/s/Samcef/Samcef-17.0-03.eb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+homepage = 'http://www.plm.automation.siemens.com/en_us/products/lms/samtech/samcef-solver-suite/'
+description = """FEM solver solution suite for basic linear structures to advanced, flexible nonlinear mechanisms and
+ thermal applications."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# licensed software, no source URls for automatic download
+# note: unpacking requires that 7z (p7zip) is available
+sources = ['SamcefV17.0-03.iso']
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/Samcef/Samcef-17.0-03.eb]: oops, hardwired version, better to avoid that!

Comment (boegel) [inline easybuild/easyconfigs/s/Samcef/Samcef-17.0-03.eb]: aah, good point, I'll get that fixed, thanks @fgeorgatos!

@wpoely86: -1 for you ;)

Comment (boegel) [inline easybuild/easyconfigs/s/Samcef/Samcef-17.0-03.eb]: see #1942

**Pull request merged**